reactDashboard/src/App.css

```diff
@@ -1,83 +1,165 @@
-.underHeatmapSelector {
-  border: 1px solid #2a84bb;
-  margin-top: 8px;
-  margin-bottom: 8px;
+.darkmode {
+  background-color: #121212;
 }
 
-.full-width-hr {
-  border: 1px solid #2a84bb;
-  margin: 10px 0;
-  padding: 0;
+.full-screen {
+  background-color: #ffffff;
 }
 
-.full-screen {
-  background-color: White;
+/* color is default white for header */
+header {
+  padding-top: 10px;
 }
 
-.link {
-  color: #3c3c3b;
-  font-weight: bold;
-  text-decoration: none;
+.darkmode header {
+  background-color: #121212;
+}
+
+/* color is default white for .region-selector-label */
+.darkmode .region-selector-label {
+  color: #f2f2f2;
+}
+
+.heatmaps-row,
+.region-page-top-row {
+  padding: 15px 10px;
+  margin: 0;
+  background-color: #f2f2f2;
+}
+
+.darkmode .heatmaps-row {
+  background-color: #262626;
+}
+
+.darkmode .region-page-top-row {
+  background-color: rgb(56, 56, 56);
+}
+
+.geo-map-column {
+  padding-left: 5px;
+  padding-right: 5px;
+  margin-bottom: 5px;
+}
+
+/* color is default white for .region-tiles-column */
```

Comment:
I get that some css declaration makes this element white by default. My suggestion was that overriding that element selector in darkmode might be cleaner than overriding all (or at least some) of the places the selector affects.

Reply:
This particular example isn't applicable because the parent background-color is the same colour as the element on top of this.